test/__snapshots__/mathml-spec.js.snap

```diff
@@ -0,0 +1,135 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`A MathML builder should generate <mphantom> nodes for \\phantom 1`] = `
+
+<math>
+  <semantics>
+    <mrow>
+      <mphantom>
+        <mi>
+          x
+        </mi>
+      </mphantom>
+    </mrow>
+    <annotation encoding="application/x-tex">
+      \\phantom{x}
+    </annotation>
+  </semantics>
+</math>
+
+`;
+
+exports[`A MathML builder should generate the right types of nodes 1`] = `
+
+<math>
+  <semantics>
+    <mrow>
+      <mi>
+        sin
+      </mi>
+      <mrow>
+        <mi>
+          x
+        </mi>
+      </mrow>
+      <mo>
+        +
+      </mo>
+      <mn>
+        1
+      </mn>
+      <mspace width="0.277778em">
+      </mspace>
+      <mtext>
+        a
+      </mtext>
+    </mrow>
+    <annotation encoding="application/x-tex">
+      \\sin{x}+1\\;\\text{a}
+    </annotation>
+  </semantics>
+</math>
+
+`;
+
+exports[`A MathML builder should make prime operators into <mo> nodes 1`] = `
+
+<math>
+  <semantics>
+    <mrow>
+      <msup>
+        <mi>
+          f
+        </mi>
+        <mrow>
+          <mo mathvariant="normal">
+            ′
+          </mo>
+        </mrow>
```

Comment:
I was surprised to see the `mo` in an `mrow`.  Otherwise this markup matches the recommended markup [described in this Mozilla post](https://bugzilla.mozilla.org/show_bug.cgi?id=140439#c7).  Is it worth checking for a size-1 `mrow` element and removing it?  (I think we do this elsewhere.)